core/src/main/scala/org/trustedanalytics/sparktk/frame/internal/ops/unflatten/UnflattenColumns.scala

```diff
@@ -0,0 +1,55 @@
+/**
+ *  Copyright (c) 2015 Intel Corporation 
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.trustedanalytics.sparktk.frame.internal.ops.unflatten
+
+import org.apache.commons.lang3.StringUtils
+import org.trustedanalytics.sparktk.frame.internal.{ FrameState, FrameTransform, BaseFrame }
+import org.trustedanalytics.sparktk.frame.internal.rdd.FrameRdd
+
+trait UnflattenColumnsTransform extends BaseFrame {
+  private val defaultDelimiter = ","
```

Comment:
better make this part of the case class instead; this is adding val defaultDelimiter to Frame  --could easily collide or be misinterpreted there.